app/src/androidTest/java/org/mozilla/reference/browser/ui/robots/SyncedTabsRobot.kt

```diff
@@ -29,15 +26,13 @@ class SyncedTabsRobot {
 
     private fun assertNotSignedInSyncTabsView() {
         assertTrue(
-            mDevice.findObject(UiSelector()
-                .resourceId("$packageName:id/synced_tabs_status"))
-            .waitForExists(TestAssetHelper.waitingTime))
+            mDevice.findObject(
+                UiSelector()
+                    .resourceId("$packageName:id/synced_tabs_status")
+                    .textContains("Connect with a Firefox Account.")
 
-        onView(
-            allOf(
-                withText(R.string.synced_tabs),
-                withText(R.string.synced_tabs_connect_to_sync_account)
-            )
+            ).waitForExists(waitingTime)
         )
+        Log.i("Andi", "Verified synced tabs")
```

Comment:
@AndiAJ Is this a debug line or actually part of the test?

Reply:
Just a debug line, sorry I've missed this